<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="QD-44" ignored="0" name="Regression01" tests="9" failures="1" timestamp="21 Mar 2018 11:23:13 GMT" time="21.152" errors="0">
  <testcase name="failed_test1" time="0.008" classname="com.qa.TEST_page.QA_Failed_testing">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at com.qa.TEST_page.QA_Failed_testing.failed_test1(QA_Failed_testing.java:17)
at java.util.concurrent.FutureTask.run(FutureTask.java:266)
at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
at java.lang.Thread.run(Thread.java:748)
... Removed 20 stack frames]]>
    </failure>
  </testcase> <!-- failed_test1 -->
  <testcase name="next_page" time="1.846" classname="com.qa.TEST_page.QA_HOME__Test"/>
  <testcase name="next_page" time="3.03" classname="com.qa.TEST_page.QA_IPAD__Test"/>
  <testcase name="next_page" time="0.508" classname="com.qa.TEST_page.QA_MAC__Test"/>
  <testcase name="next_page" time="1.492" classname="com.qa.TEST_page.QA_IPHONE__Test"/>
  <testcase name="next_page" time="1.569" classname="com.qa.TEST_page.QA_WATCH__Test"/>
  <testcase name="next_page" time="1.396" classname="com.qa.TEST_page.QA_TV__Test"/>
  <testcase name="next_page" time="2.668" classname="com.qa.TEST_page.QA_MUSIC__Test"/>
  <testcase name="next_page" time="1.842" classname="com.qa.TEST_page.QA_SUPPORT__Test"/>
</testsuite> <!-- Regression01 -->
